deployment/aerie_db_migration.py

```diff
@@ -349,6 +349,12 @@ def __apply_workspaces_migration__(self) -> bool:
     if results.pop(0)[0] != 'id':
       exit_with_error("Error while fetching user sequences from the database.")
 
+    # admin headers for making requests to the workspace service without a JWT
+    workspace_service_headers = {
+      "x-hasura-admin-secret": self.admin_secret,
+      "x-hasura-role": "admin",
```

Comment:
Typo: `aerie_admin` is the name of Aerie's admin role, not `admin`:
```suggestion
      "x-hasura-role": "aerie_admin",
```